expr/test.go

```diff
@@ -9,6 +9,26 @@ import (
 
 var results []models.Series
 
+func equalTags(exp, got []models.Series) error {
+	for i, g := range got {
+		o := exp[i]
+		if len(g.Tags) != len(o.Tags) {
+			return fmt.Errorf("series %d: len tags expected %d, got %d", i, len(o.Tags), len(g.Tags))
+		}
+		for k, v := range g.Tags {
```

Comment:
I really like https://github.com/google/go-cmp and think we should use it in more places.
e.g.
```
func TestHelloWorld(t *testing.T) {

        want := map[string]string{
                "foo": "bar",
        }
        got := map[string]string{
                "foo":   "bar",
                "extra": "hello",
        }

        if diff := cmp.Diff(want, got); diff != "" {
                t.Errorf("mismatch (-want +got):\n%s", diff)
        }
}
~

go test -v .                                                                                                                                                         ⏎
=== RUN   TestHelloWorld
    TestHelloWorld: main_test.go:20: mismatch (-want +got):
          map[string]string{
        + 	"extra": "hello",
          	"foo":   "bar",
          }
--- FAIL: TestHelloWorld (0.00s)
FAIL
FAIL	github.com/Dieterbe/sandbox/cmp_map	0.002s
FAIL
```

Reply:
nifty!